### PR TITLE
Remove the ability to cancel all "in progress" tasks

### PR DESCRIPTION
**Pull request by peppy**
Brought up by @nekodex.

I don't think anyone would want to do this. It could only be a misclick. For ongoing operations, a user should have to cancel each one manually.

### CODE REVIEW

**Comment by bdach** [comment]
I don't know about this.

- First of all, being able to cancel all in progress tasks can actually come in handy sometimes. Example: when downloading beatmaps via listing, every single beatmap download spawns a progress notification. If you now want to cancel all the downloads for whatever reason, you'd have to cancel every single progress notification. To me that feels worse than what we already have.
- Secondly, after the tasks are cancelled, they stay there on the notification overlay. Previously the "cancel all" button would also dismiss all already-cancelled notifications. Now that the button isn't there anymore, they'd have to dismiss all of them manually by hand.

I'd argue that at the least the button should stay, but rather than cancel all tasks _and_ dismiss already-cancelled ones, it should only dismiss the already-cancelled ones. Doing this to bypass problems with layered sample playback feels like pushing it a bit, I feel that the button has legitimate use cases.

**Comment by peppy** [comment]
That's a fair call. I think we can always display the "Clear all" text and only have it clear things not in progress. It's a pretty standard behaviour.

File: osu.Game/Overlays/Notifications/NotificationSection.cs [hunk 1]
```diff
@@ -75,7 +73,7 @@ private void load(OsuColour colours)
                     {
                         new ClearAllButton
                         {
-                            Text = clearButtonText,
+                            Text = NotificationsStrings.ClearAll,
```
Comment (Joehuu) [inline osu.Game/Overlays/Notifications/NotificationSection.cs:76]: Was the removal of `ToUpper()` intentional?

Comment (peppy) [reply osu.Game/Overlays/Notifications/NotificationSection.cs:76]: likely not

**Review by bdach — APPROVED** [review]

**Pull request merged**